js/os.ts

```diff
@@ -23,6 +23,41 @@ interface CodeInfo {
   sourceMap: string | undefined;
 }
 
+interface IsTTY {
+  stdin: boolean;
+  stdout: boolean;
+  stderr: boolean;
+}
+
+let _isTTY: IsTTY; // cache
+
+/** Check if running in terminal.
+ *
+ * isTTY().stdin; // stdin
+ * isTTY().stdout; // stdout
+ * isTTY().stderr; // stderr
```

Comment:
Example code should be indented, and it should have an import, so people can test by copy & paste
```js
import { isTTY } from "deno";
console.log(isTTY.stdout);
```